### PR TITLE
Revamp. Also fixed DOCS-8375.

### PR DESCRIPTION
**Pull request by hannanelson**
[19-after-Manage Schedules _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-runtime-manager/files/4893742/19-after-Manage.Schedules._.MuleSoft.Documentation.pdf)

### CODE REVIEW

File: modules/ROOT/pages/hybrid-schedule-mgmt.adoc [hunk 1]
```diff
@@ -1,37 +1,99 @@
...
 
+== Considerations and Limitations
+
+When using the *Schedules* tab, consider the following:
+
+* If a scheduled job is not triggered because the application is not running,
+Runtime Manager triggers the job as soon as the application restarts.
+* When a schedule is used to trigger a flow with subflow in a Mule 4.x application, it waits until both main flow and subflow complete before scheduling the next run.
++
+*Is this true? I found it in the CloudHub Schedule info.*
```
Comment (marialuzshin) [inline modules/ROOT/pages/hybrid-schedule-mgmt.adoc]: No sure about this, we need to validate it.

Comment (hannanelson) [reply modules/ROOT/pages/hybrid-schedule-mgmt.adoc]: Removed for now

**Comment by hannanelson** [comment]
Consolidated hybrid-schedule-mgmt-config into hybrid-schedule-mgmt, removed -config, added page-alias.

**Review by marialuzshin — DISMISSED** [review]
LGTM!

File: modules/ROOT/pages/hybrid-schedule-mgmt.adoc [hunk 2]
```diff
@@ -1,37 +1,96 @@
...
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
-image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
+You can use Runtime Manager to view and control the Scheduler components within the flows in your deployed applications.
+Use the *Schedules* tab in the left navigation menu to enable or disable Scheduler elements,
+change the frequency, or run the scheduled job immediately,
+all without changing the running application.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/hybrid-schedule-mgmt.adoc:17]: Nice intro!

File: modules/ROOT/pages/hybrid-schedule-mgmt.adoc [hunk 3]
```diff
@@ -1,37 +1,96 @@
...
-You can use Runtime Manager to view and control polling elements within the flows of your deployed applications. Runtime Manager automatically detects the polling elements within your flows. These elements are listed on the *Schedulers* tab of the *Manage Applications* page.
-
-Runtime Manager lists the name of each poll element as `polling://{flow_name}`.
+From the *Schedules* tab, you can:
 
-From this tab, you can perform the following:
+* Enable or disable the Scheduler
+* Change the frequency
+* Specify when to start running the Scheduler
+* Specify the timezone (for a cron Scheduler)
```
Comment (kbostrom) [inline modules/ROOT/pages/hybrid-schedule-mgmt.adoc]:
```suggestion
* Specify the timezone for a cron Scheduler
```

File: modules/ROOT/pages/hybrid-schedule-mgmt.adoc [hunk 4]
```diff
@@ -1,37 +1,96 @@
...
+image::hybrid-scheduler-cron.png[Schedule configuration in the details pane]
+
+** Update a fixed-frequency Scheduler:
++
+.The arrow shows the schedule configuration in the details pane.
+image::hybrid-scheduler-fixed-frequency.png[Schedule configuration in the details pane]
+
+--
+. Click *Update* to save changed settings.
+. Select multiple checkboxes to enable and disable multiple schedules.
```
Comment (kbostrom) [inline modules/ROOT/pages/hybrid-schedule-mgmt.adoc]: I feel like this belongs up with the step 
. Select the checkbox to manage a single Scheduler:
Maybe 
. Either select one checkbox to manage a single Scheduler, or select multiple checkboxes to enable and disable (but not update) multiple schedules:

Comment (kbostrom) [reply modules/ROOT/pages/hybrid-schedule-mgmt.adoc]: Or we could make each goal a separate step?
. Enable one or more schedules by
. Update a cron Scheduler by
. Update a fixed-frequency Scheduler by
. Disable one or more schedules by

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by kbostrom — APPROVED** [review]
Reapproval

**Pull request merged**